python/dgl/data/chem/tox21.py

```diff
@@ -0,0 +1,31 @@
+import numpy as np
+
+from .csv_dataset import CSVDataset
+from .utils import smile2graph
+from ..utils import get_download_dir, download, _get_dgl_url, Subset
+
+try:
+    import pandas as pd
+except ImportError:
+    raise ImportError("Please install pandas")
+
+
+class Tox21(CSVDataset):
+    _urls = 'dataset/tox21.csv.gz'
+
+    def __init__(self, smile2graph=smile2graph):
+        data_path = get_download_dir() + '/tox21.csv.gz'
+        download(_get_dgl_url(self._urls), path=data_path)
+        df = pd.read_csv(data_path)
+        df = df.drop(columns=['mol_id'])
```

Comment:
I don't think we should drop them.

Reply:
Added `self.id=df['mol_id']`. Because some csv dataset may not have id column, therefore I assume the dataframe passed to CSVDataset is already preprcessed with only smiles and labels.

Reply:
I understand that, but that part might still be useful for some users.

Reply:
You can maintain the id in the inherited class. It's still there.

Reply:
I'm fine with additional `self.id`, just not to completely remove them.